.github/workflows/go.yaml

```diff
@@ -100,8 +110,8 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        go-version: ["1.21"] # Disable Go 1.22 until TinyGo releases update
-        tinygo-version: ["0.30.0"]
+        go-version: ["1.21", "1.22"]
+        tinygo-version: ["0.31.2"]
```

Comment:
Can these be pulled out into YAML references so we don't forget to update them in both places?

Reply:
Filed this in #973, "great idea, let's do it later"